[airflow-216] Add Sqoop Hook and Operator

jwi078 opened the pull request:
Please accept this PR that addresses the following issues:
https://issues.apache.org/jira/browse/AIRFLOW-216

This patch adds a Sqoop hook and operator that implements Sqoop import
and export for Sqoop 1. The commit is in respective directory within
contrib folder. The hook is a wrapper around the sqoop 1 binary
and based upon Bolke De Bruin's implementation.

bolkedebruin commented on airflow/contrib/operators/sqoop_operator.py:
```diff
@@ -0,0 +1,78 @@
+from airflow.models import BaseOperator
```
Please add license header

bolkedebruin commented on airflow/contrib/operators/sqoop_operator.py:
```diff
@@ -0,0 +1,78 @@
+
+        """
+        :param conn_id: str
+	:param type_cmd: str specify command to execute "export" or "import"
```
missing space

bolkedebruin commented on airflow/contrib/operators/sqoop_operator.py:
```diff
@@ -0,0 +1,78 @@
+        self.export_dir=export_dir
+
+    def execute(self, context):
+
```
too many white lines

bolkedebruin commented on airflow/contrib/operators/sqoop_operator.py:
```diff
@@ -0,0 +1,78 @@
+
+        hook = SqoopHook(
+                conn_id=self.conn_id)
+
```
too many white lines

bolkedebruin commented on airflow/contrib/hooks/sqoop_hook.py:
```diff
@@ -0,0 +1,229 @@
+        cmd = self._export_cmd(export_dir,num_mappers)
+
+        cmd += ["--table", table]
+
```
too many white lines

bolkedebruin commented on airflow/contrib/hooks/sqoop_hook.py:
```diff
@@ -0,0 +1,229 @@
+            num_mappers = 1
+
+        cmd += ["--num-mappers", str(num_mappers)]
+
```
too many white lines

bolkedebruin commented on airflow/contrib/hooks/sqoop_hook.py:
```diff
@@ -0,0 +1,229 @@
+
+    def export_table(self, table, export_dir ,
+                     num_mappers=None):
+
```
too many whitelines

jwi078 committed:
[AIRFLOW-216] Add Sqoop Hook and Operator

This patch adds a Sqoop hook and operator that implements Sqoop import.
The hook is a wrapper around the sqoop 1 binary.
